charts/argo-workflows/values.yaml

```diff
@@ -12,6 +12,10 @@ init:
 
 createAggregateRoles: true
 
+# Override app name
+# Internally, this will be truncated at 63 chars because some Kubernetes name fields are limited to this (by the DNS naming spec).
+fullnameOverride: ""
+
```

Comment:
This is used below. 🙋 
https://github.com/argoproj/argo-helm/blob/master/charts/argo-workflows/templates/_helpers.tpl#L24-L39